packages/e2e-test-utils-playwright/src/metrics/index.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * External dependencies
+ */
+import type { Page } from '@playwright/test';
+import * as lighthouse from 'lighthouse/core/index.cjs';
+
+export class Metrics {
```

Comment:
Curious why you implemented this as a class? Not opposed, but the class methods could also be individual functions, since they don't really have anything in common. I rarely see class usage in JS these days in favor of individual functions, so I wonder why here you went a different route.

Reply:
This is how it‘s done with Playwright fixtures. Please check out the other utils in the package and the Playwright docs at https://playwright.dev/docs/test-fixtures